### ISSUE TITLE
Your current PostCSS version is 6.0.2, but postcss-ant uses 5.2.17

### ISSUE THREAD

**Issue by corysimmons** [corysimmons/postcss-ant#14]
I hate trying to use PostCSS as a preprocessor... SugarSS sucks. PreCSS sucks. Everything sucks. Just use node-sass or stylus, and chokidar or something. ffs..

**Comment by Alaev** [comment]
+1 with Cory on that. 
At the moment I use stylus.

**Comment by toklok** [comment]
How does this work to get two versions of PostCSS?  I need to run PostCSS version 6.0.16 and 5.2.17?

**Comment by corysimmons** [comment]
@toklok I don't think you can use the same API from different versions of the same package within the same project.

For instance if I make a pkg called `foo` and you `npm i foo@1` and `npm i foo@2` it will overwrite `package.json`. You _could_ fork PostCSS and postcss-ant, and rename a ton of stuff if you had no other option, but I wouldn't recommend it.

Just update PostCSS and I'll get around to updating postcss-ant whenever people complain about it.

**Comment by alive-jekyll** [comment]
...
> 
> Just update PostCSS and I'll get around to updating postcss-ant whenever people complain about it.

so I guess this error is not getting a fix - "Your current PostCSS version is 7.0.5, but postcss-ant uses 5.2.18 "?

**Comment by corysimmons** [comment]
Yeah, sorry. Haven't updated this in forever. Let me do some chores and I'll take a look.
